packages/preview/riemann/0.1.0/README.md

```diff
@@ -0,0 +1,19 @@
+# Riemann sum graphs in Typst with CeTZ
+
+A package to draw Riemann sums (and their plots) of a function with CeTZ.
+
+Usage example and docs: [manual.pdf](https://github.com/ThatOneCalculator/riemann-sum-typst-cetz/blob/main/manual.pdf)
+
+On Typst v0.6.0+ you can import the `riemann` package:
```

Comment:
just wondering
couldn't we expect the users to have this version?

if they don't the `riemann.typ` file also won't work because it uses `#import "@preview/cetz:0.1.2"` btw